pkg/virt-controller/watch/vm.go

```diff
@@ -793,6 +793,7 @@ func (c *VMController) setupVMIFromVM(vm *virtv1.VirtualMachine) *virtv1.Virtual
 	vmi.ObjectMeta.GenerateName = ""
 	vmi.ObjectMeta.Namespace = vm.ObjectMeta.Namespace
 	vmi.Spec = vm.Spec.Template.Spec
+	vmi.Status.Phase = virtv1.Pending
```

Comment:
Should this go into the VMI mutation webhook instead?

Reply:
Good point, fixed.